### DISCUSSION TITLE
How to scrape Azure resource count limits and quotas for different resource types?

### DISCUSSION THREAD

**Discussion by kaustubhdpatil** [tomkerkhove/promitor#1877] [Q&A] [answered]
We are evaluating Promitor to get the list of metrics for the Azure resources that are listed by the Azure Monitoring APIs. Although was wondering if there is a way to scrape the used count and the capacity by subscription for various Azure resources. Following are some of the resources we plan to get these counts for:
- SQL Servers per subscription
- SQL Databases per server

Is there a way to monitor this kind of metric using Promitor? I was not sure if there was already a way to scrape this information. Or is this something that is considered to be out of scope for Promitor?

**Comment by tomkerkhove** [comment]
Are you asking to scrape the resource quotas for a given service - Is that a good summary?

**Reply by kaustubhdpatil** [reply]
@tomkerkhove sorry for the late response. But yes more or less the resource quotas. For example the following output that shows the limit for the storage accounts and the number of accounts added so far -
````JSON
{
  "value": [
    {
      "unit": "Count",
      "currentValue": 55,
      "limit": 250,
      "name": {
        "value": "StorageAccounts",
        "localizedValue": "Storage Accounts"
      }
    }
  ]
}
````
As these metrics also would need to be monitored. Is there a way to achieve this in Promitor?

**Reply by tomkerkhove** [reply]
Not today but definitely a great feature to add! Would you mind opening a feature request for this?

Great idea!

**Reply by kaustubhdpatil** [reply]
Sure, will do so. thanks for the quick response.

**Reply by kaustubhdpatil** [reply]
Thank you for the response and help @tomkerkhove 
I am going to mark this as answered as the new feature request has been created - [Feature 1888](https://github.com/tomkerkhove/promitor/issues/1888)

**Reply by tomkerkhove** [reply]
Thank you!

**Comment by kaustubhdpatil** [comment, marked as answer]
Feature created. For reference to the feature check the last comment.
